### DISCUSSION TITLE
Amount of RAM needed for teslamate

### DISCUSSION THREAD

**Discussion by billerby** [teslamate-org/teslamate#1841] [Getting started]
Totally new to teslamate and tried setting it up (the advanced with an apache acting as a reverse proxy) on a aws lightsail instance yesterday (the cheapest one with only 512 MB:s of RAM). That failed when running docker-compose, got "killed" written to the console and watching with top in another terminal made it quite appearant that all RAM was used. Tried searching through the discussions and docs but can not find any recommendations on of much RAM is needed. Anyone know?

Regards!
/Erik

**Comment by DrMichael** [comment]
CONTAINER ID   NAME                    CPU %     MEM USAGE / LIMIT   MEM %     NET I/O          BLOCK I/O         PIDS
addff3bc5f9a   teslamate_grafana_1     0.06%     41.71MiB / 4GiB     1.02%     54.5MB / 81MB    4.69GB / 0B       13
0131b49834c7   teslamate_teslamate_1   0.18%     164.9MiB / 4GiB     4.03%     123MB / 197MB    4.33GB / 16.4kB   26
98f92b5f775f   teslamate_mosquitto_1   0.05%     1.797MiB / 4GiB     0.04%     5.8MB / 2.93MB   2.36GB / 0B       1
6458cc73cc49   teslamate_pgadmin_1     0.06%     125.4MiB / 4GiB     3.06%     735kB / 879kB    3.44GB / 2.24MB   11
ea09e72cd1c0   teslamate_database_1    1.29%     227.6MiB / 4GiB     5.56%     186MB / 139MB    6.48GB / 461MB    19

512MB is not enough. With 1GB it is running fine.
